Source/Core/Core/IOS/Network/KD/NetKDRequest.cpp

```diff
@@ -17,6 +17,7 @@
 #include "Common/Logging/Log.h"
 #include "Common/NandPaths.h"
 #include "Common/SettingsHandler.h"
+#include "Common/StringUtil.h"
```

Comment:
In case anyone else wonders about this, it's for the preexisting call to Common::BytesToHexString.